### ISSUE TITLE
How can I add registrant to a single occurrence id of the meeting

### ISSUE THREAD

**Issue by aneeshtan** [MacsiDigital/laravel-zoom#14]
Hi, 

This is the easy way to add registrant to a meeting, but what if we a recurring meeting and we want to add registrant to a single occurrence_id ?

```
$registrant = $meeting->registrants()->create([
    	'email' => 'registratn@domain.com',
    	'first_name' => 'Test',
    	'last_name' => 'Registrant'
    ]);
```

**Comment by colinhall17** [comment]
The save method at present does not insert a query string to the url - to filter by occurrence it would need one as per the documentation

**Comment by colinhall17** [comment]
Hi all

Apologies in the delay in coming back to you, when I created this package it was a quick build for a project I needed it for and I made it open source in case it would help someone. 

With the increased use of Zoom around the world its gaining some traction, so I've built 3 new versions for Laravel 5.5 upwards, make sure you select the correct version for your version of Laravel.

The new version now allows you to attach registrants to occurrences, the documentation is a lot more detailed and states how this can be done.

Open a new issue with any problems.
